src/components/skip-to-content/skip-to-content.js

```diff
@@ -2,6 +2,7 @@ export default function skipToMain(link) {
   const id = link.getAttribute('href').replace('#', '');
   link.addEventListener('click', event => {
     event.preventDefault();
-    document.getElementById(id).scrollIntoView();
+    document.getElementById(id).tabIndex = 0;
+    document.getElementById(id).focus();
```

Comment:
What happens if there is no input on the page? `scrollIntoView` works on any element with the provided `id` but `focus` relies on a focusable element being the target.

Reply:
Sorry, I see you are giving a tab order to the element to allow focus.